apps/explorer/lib/explorer/chain.ex

```diff
@@ -4612,19 +4613,19 @@ defmodule Explorer.Chain do
     |> limit(^paging_options.page_size)
   end
 
-  defp handle_random_access_paging_options(query, empty_options) when empty_options in [nil, [], %{}],
+  defp handle_random_access_paging_options(query, empty_options, _is_extra_transfer) when empty_options in [nil, [], %{}],
```

Comment:
Another point regarding the extra parameter: while currently, random access paging is used mostly for paginating through txs, these methods are written to be generic.

Reply:
please see my latest push, other methods calling it will not have to add the boolean, only those that want to set it as true.